scripts/generator/templates/template-vaadin-bom.xml

```diff
@@ -115,6 +115,13 @@
                 <artifactId>jna-platform</artifactId>
                 <version>${jna.version}</version>
             </dependency>
+
+            <!-- workaround,should be removed after this module has been added to hilla-bom -->
+            <dependency>
+                <groupId>com.vaadin.hilla</groupId>
+                <artifactId>hilla-dev-mode</artifactId>
```

Comment:
Maybe I missed something important, but how with this structure one can exclude all development related dependencies in hybrid app ? I expected only one step as previously - excluding `vaadin-dev` in pom.xml.
See my comment [here](https://github.com/vaadin/platform/issues/4946#issuecomment-1914431757).

Reply:
this is only a bom, which doesn't bring in any dependency directly, as it is only under dependencyManagement. 

also for this case, it is a workaround as `hilla-dev-mode` is not in `hilla-bom` yet

Reply:
Yep, agree, maybe this isn't a better place to ask this.

Reply:
However, not clear to me how Hilla dev things should be excluded in prod mode, and should they even be excluded?